src/triple.rs

```diff
@@ -401,7 +401,8 @@ impl Triple {
 #[macro_export]
 macro_rules! triple {
     ($str:tt) => {
-        target_lexicon::Triple::from_str($str).expect("invalid triple literal")
+        <target_lexicon::Triple as core::str::FromStr>::from_str($str)
```

Comment:
```suggestion
        <$crate:Triple as core::str::FromStr>::from_str($str)
```
To allow the user to rename target-lexicon.